README.md

```diff
@@ -1,96 +1,209 @@
+[Logseq](https://logseq.com) stores your notes locally on your device in plain text files, ensuring that your data is **private** and **secure**. Logseq also strongly emphasizes [**user control**](https://www.gnu.org/philosophy/free-sw.en.html), allowing you to access and manage your notes confidently.
 
-In the unlikely event that the website is down or cannot be maintained, your data is, and will always be yours.
+In addition to its core features, Logseq has a growing ecosystem of plugins and themes that enable a wide range of workflows and customization options. Mobile apps are also available, providing access to most features from the desktop application.
 
-![Image of logseq](https://cdn.logseq.com/%2F8b9a461d-437e-4ca5-a2da-18b51077b5142020_07_25_Screenshot%202020-07-25%2013-29-49%20%2B0800.png?Expires=4749255017&Signature=Qbx6jkgAytqm6nLxVXQQW1igfcf~umV1OcG6jXUt09TOVhgXyA2Z5jHJ3AGJASNcphs31pZf4CjFQ5mRCyVKw6N8wb8Nn-MxuTJl0iI8o-jLIAIs9q1v-2cusCvuFfXH7bq6ir8Lpf0KYAprzuZ00FENin3dn6RBW35ENQwUioEr5Ghl7YOCr8bKew3jPV~OyL67MttT3wJig1j3IC8lxDDT8Ov5IMG2GWcHERSy00F3mp3tJtzGE17-OUILdeuTFz6d-NDFAmzB8BebiurYz0Bxa4tkcdLUpD5ToFHU08jKzZExoEUY8tvaZ1-t7djmo3d~BAXDtlEhC2L1YC2aVQ__&Key-Pair-Id=APKAJE5CCD6X7MP6PTEA)
+Whether you're a student, a professional, or anyone who values a clear and organized approach to managing your ideas and notes, Logseq is an excellent choice for **improving your knowledge management** and collaboration skills.
 
-## Sponsors
+![Image of Logseq](https://user-images.githubusercontent.com/25513724/220209160-e8ce0f69-5ef4-49d6-be4d-7aa91accbc5b.png)
+
+<a href="https://github.com/logseq/logseq/releases/latest/">
+        <img src="https://img.shields.io/badge/Download_Logseq-100000?style=for-the-badge&logo=flatpak&logoColor=white&labelColor=002b36&color=85c8c8"
+            align="right"
+            alt="Download Logseq"/></a>
+
+## 👀 How can I use it?
+
+To start using Logseq, follow these simple steps:
+
+1. [Download](https://github.com/logseq/logseq/releases/latest) the latest version of Logseq.
+2. Install Logseq on your device and launch the application.
+3. Start writing and organizing your notes, tasks, and ideas in Logseq.
+
+That's it! You can now enjoy the benefits of using Logseq to streamline your workflow, manage your projects, and stay on top of your goals. Have fun! 🎉
+
+> **Note**
+> Checkout our docs at [docs.logseq.com](https://docs.logseq.com/#/page/Contents)
+
+## 🫶 Support Logseq Development
+
+If you find Logseq useful and want to help us keep the project growing, please consider supporting our contributors on [Open Collective](https://opencollective.com/logseq). Your support is a way of showing our contributors that their efforts are appreciated and motivates them to continue their excellent work. Every contribution, no matter how small, helps us to keep improving Logseq.
```

Comment:
helps us to keep -> helps us keep

Reply:
motivates them -> motivating them

Reply:
> helps us to keep -> helps us keep

Both are correct. Will update to the shorter version.

> motivates them -> motivating them

This is an incorrect suggestion

> Your support **shows** our contributors that their efforts are appreciated and **motivates** them to continue their excellent work.

**`shows`** `and` **`motivates`**